### PR TITLE
build(cordova/apple/macos): signing reset and version bump

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist [hunk 1]
```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.7.0</string>
```
Comment (fortuna) [inline src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist:20]: Don't you need to change Outline-Info too?

Comment (fortuna) [inline src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist:20]: We never released 1.7.1, so we could keep that number.

Comment (fortuna) [reply src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist:20]: Same on macOS

Comment (daniellacosse) [reply src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist:20]: It doesn't compile unless the versions match

Comment (daniellacosse) [reply src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist:20]: It was already changed, it just wouldn't compile 😅

**Review by fortuna — APPROVED** [review]

**Pull request merged**